data-archive/db_scripts/mosip_archive/ddl/archive-credential-credential_transaction.sql

```diff
@@ -15,7 +15,7 @@
 -- object: archive.mosip_credential_credential_transaction | type: TABLE --
 -- DROP TABLE IF EXISTS archive.mosip_credential_credential_transaction CASCADE;
 CREATE TABLE archive.mosip_credential_credential_transaction(
-	id character varying(36) NOT NULL,
+	id character varying(64) NOT NULL,
```

Comment:
These kind of table structure changes will keep happening in every projects, can we not somehow get this structure from the from the source table itself ?

Reply:
To be taken care with the new design of getting the ddl from the image or github instead of maintaining the copy.